### PR TITLE
MULTIARCH-4563: Disable branch protection

### PR DESCRIPTION
**Pull request by AnnaZivkovic**
Disable branch protection in the prow config for the MTO operator so that non-prow, non-required jobs do not block tide from merging Pull Requests

### CODE REVIEW

File: core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+branch-protection:
+  orgs:
+    openshift:
+      repos:
+        multiarch-tuning-operator:
+          branches:
+            main:
+              protect: true
```
Comment (aleskandro) [inline core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml:8]: shouldn't we set this to false?

Comment (AnnaZivkovic) [reply core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml]: That would make scene, but the task said set it to true

Comment (Prashanth684) [reply core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml]: i assume this is a temporary measure?

Comment (aleskandro) [reply core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml]: Ahah my bad then 😅


Yes, we temporarily need to be able to push on main and overcome the failures in Konflux as they block us from merging and we always need to override

**Review by aleskandro — APPROVED** [review]
Thanks @AnnaZivkovic

**Pull request merged**